### DISCUSSION TITLE
locale in argv.json not work?

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5644] [Q&A]
install code-server `v4.7.1` and install cn-langauge, but visit web, not display cn. I have to click `set display languge` manualy.

so locale in argv.json not work?

I want locale control in setting.json or argv.json.

![image](https://user-images.githubusercontent.com/9245110/195606689-f4f1e0e3-1d67-4625-9c3a-21e63ad8fd82.png)

![image](https://user-images.githubusercontent.com/9245110/195606931-388747b4-2e6f-4f25-be7f-161024a7e728.png)

## about code 

```
code-server --install-extension ms-ceintl.vscode-language-pack-zh-hans --extensions-dir $ROOT_PATH/.local/share/extensions


cat << EOF >  $ROOT_PATH/.local/share/User/argv.json
{"locale": "zh-cn"}
EOF

```

**Comment by jsjoeio** [comment]
I think this is a bug in code-server. Someone posted a hint [here](https://github.com/coder/code-server/issues/3372#issuecomment-1029340034) but we haven't had time to fix it.

**Reply by alanhe421** [reply]
which old version is ok?

**Reply by jsjoeio** [reply]
Hmm...I'm not sure? Maybe before v4?

**Reply by alanhe421** [reply]
> Hmm...I'm not sure? Maybe before v4?

I confirmed that `v3.12.0` is ok.
